src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -694,6 +694,31 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 		const struct xccdf_version_info *xccdf_version = xccdf_benchmark_get_schema_version(benchmark);
 		const char *xccdf_version_name = xccdf_version_info_get_version(xccdf_version);
 
+		if (NULL != profile_description) {
+			unsigned int newLines = 0;
+			const char *c = profile_description;			
+			while (*c != '\0') {
+				if (*c == '\n')
+					++newLines;
+				++c;
+			}
+			
+			if (newLines > 0) {
+				unsigned int description_length = c - profile_description;
```

Comment:
Isn't this the same as strlen(profile_description)? I know this is slightly faster but it's also unreadable. It can also be const and `size_t`.

Reply:
strlen implies one more pass through the string and I tried to avoid that, however, I simplified the code and got rid of it.